### DISCUSSION TITLE
2019/how-to-download-an-m3um3u8-playlist-stream-without-a-special-software/

### DISCUSSION THREAD

**Discussion by giscus** [hellt/netdevops.me#21] [Show and tell]
# 2019/how-to-download-an-m3um3u8-playlist-stream-without-a-special-software/

Applying DevOps to networks.

https://netdevops.me/2019/how-to-download-an-m3um3u8-playlist-stream-without-a-special-software/

**Comment by PERIX4460** [comment]
VLC doesn't seem to be able to open the link, is it a problem with the link or am i doing something wrong?
